apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/config/SvAppConfig.scala

```diff
@@ -267,6 +267,7 @@ case class SvAppBackendConfig(
     expectedTaskDuration: Long = 5000, // milliseconds
     expiredRewardCouponBatchSize: Int = 100,
     bftSequencerConnection: Boolean = true,
+    skipSynchronizerInitialization: Boolean = false,
```

Comment:
I tried to build an option that just handles the sequencer but mediator and sequencer are too closely coupled so this seems like the better option.


I think eventually we might also want to change the default to true. But I think for the next release having an option that SVs need to turn on is good enough.